lib/backends/vault-backend.js

```diff
@@ -55,10 +57,16 @@ class VaultBackend extends KVBackend {
     const secretResponse = await this._client.read(key)
 
     if (kvVersion === 1) {
+      if (keyOptions && keyOptions.isBinary) {
+        secretResponse.data[property] = Buffer.from(secretResponse.data[property], 'base64').toString('utf8')
```

Comment:
This feels kind of scary 😄 I guess its fine to turn it into a string but feels like we have an underlying problem in the code forcing us to write weird code :)
Main issue is probably that the KV backend expects us to return a string in `_get` forcing us to serialise here when we already have an object.